menu.json

```diff
@@ -1,18 +1,19 @@
 [
-  "Install npm",
-  "Dev Environment",
-  "Install A Module",
-  "Listing Dependencies",
-  "npm Test",
-  "Login",
-  "Package Niceties",
-  "Publish",
-  "Version",
-  "Publish Again",
-  "Dist Tag",
-  "Dist Tag Removal",
-  "Outdated",
-  "Update",
-  "Rm",
-  "Finale"
+  "00 Install npm",
```

Comment:
By changing this you should also change the i18n/en.json in which there are pointers to these names